content/3_Silver/Prefix_Sums.mdx

```diff
@@ -395,6 +395,19 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```python
+def psum(a):
+	psum=[0]
+	for i in a:
+		psum.append(psum[-1]+i)
```

Comment:
add comment saying psum[-1] is last element?
or honestly, i think it's better to just write psum[i+1]=psum[i]+a[i]

also typo, should be a[i], not i

Reply:
JK just noticed it's `for i in a`, myb

Reply:
PYTHON